Automattic/woocommerce-payments
# Display transaction fee structure in timeline

Fixes #954

**Server PR:** 434-gh-Automattic/woocommerce-payments-server

#### Changes proposed in this Pull Request

Use the `fee_rates` property of the captured timeline event to display the fee structure (percentage + fixed amount).

<img width="1063" alt="image" src="https://user-images.githubusercontent.com/5311119/99072770-5761b580-25bd-11eb-9276-9b396cb9bc00.png">

In case the property is not available, the timeline will fall back to the standard amount-only format. This will be helpful both before the server PR gets merged, and if we decide to change the format of the fee structure in the future.

#### Testing instructions

1. Check out the server PR.
2. Go to a transaction details screen. You should see the fixed and percentage amounts as in the screenshot above.

-------------------
